build_tools/bump_submodules.py

```diff
@@ -130,8 +204,25 @@ def main(argv):
         action=argparse.BooleanOptionalAction,
         help="Pin composable_kernel to version tagged in MIOpen",
     )
+    parser.add_argument(
+        "--components",
+        type=str,
+        nargs="+",
+        default="default",
+        help="""List of components (subdirectories) to bump. Choices:
+                  default,
+                  base,
```

Comment:
You could add a docstring comment to the top of this file and/or some notes at https://github.com/ROCm/TheRock/blob/main/docs/development/git_chores.md showing an example of how to run the script with this argument.